main.c

```diff
@@ -861,8 +882,10 @@ main (int   argc,
   GStrv static repos = NULL;
   GStrv static exclude_packages = NULL;
   static gboolean verbose = FALSE;
+  static gboolean test = FALSE;
   static const GOptionEntry opts[] = {
     { "verbose", 'v', 0, G_OPTION_ARG_NONE, &verbose, "Show extra debugging information", NULL },
+    { "test", 't', 0, G_OPTION_ARG_NONE, &test, "Run with test repo", NULL},
```

Comment:
ideally we would turn fus into some kind of a library and create special test program, but I don't care much about this now. I would probably prefer if you compile fus with `-DFUS_TESTING`, then it would be just different program.